server/jetstream_cluster_4_test.go

```diff
@@ -2549,3 +2549,294 @@ func TestJetStreamClusterMetaSyncOrphanCleanup(t *testing.T) {
 		require_Equal(t, state.Msgs, 10)
 	}
 }
+
+func TestJetStreamConsumeWithStartSequence(t *testing.T) {
```

Comment:
```suggestion
func TestJetStreamClusterConsumeWithStartSequence(t *testing.T) {
```

Reply:
This test runs both single server and cluster, so I didn't name it `Cluster`. Do you think I should add it?

Reply:
the CI test script for this file only matches TestJetStreamCluster

Reply:
Gotcha. Thank you.